pyro/contrib/epidemiology/util.py

```diff
@@ -106,16 +106,19 @@ def align_samples(samples, model, particle_dim):
 W16 = numpy.array(W16)
 
 
-def compute_bin_probs(s, num_quant_bins=3):
+def compute_bin_probs(s, num_quant_bins):
     """
     Compute categorical probabilities for a quantization scheme with num_quant_bins many
     bins. `s` is a real-valued tensor with values in [0, 1]. Returns probabilities
     of shape `s.shape` + `(num_quant_bins,)`
     """
-    if num_quant_bins not in [4, 8, 12, 16]:
-        raise ValueError("Supported quantization strategies have 4, 8, 12, or 16 bins")
 
     t = 1 - s
+
+    if num_quant_bins == 2:
```

Comment:
why not the smoother cubic version?

Reply:
I think my/our early attempt at a 2-point cubic spline was ill-conceived and not very smooth:
![image](https://user-images.githubusercontent.com/648532/83179515-ff022e80-a0d6-11ea-92a7-e349b1837957.png)